experimental/segmenter/benches/bench.rs

```diff
@@ -18,13 +18,13 @@ fn line_break_iter_latin1(c: &mut Criterion) {
     let mut group = c.benchmark_group("Line Break/Latin1");
 
     let segmenter =
-        LineSegmenter::try_new_unstable(&icu_testdata::unstable()).expect("Data exists");
+        LineSegmenter::try_new_dictionary_unstable(&icu_testdata::unstable()).expect("Data exists");
 
     let mut options = LineBreakOptions::default();
     options.line_break_rule = LineBreakRule::Anywhere;
     options.word_break_rule = WordBreakRule::BreakAll;
     let segmenter_css =
-        LineSegmenter::try_new_with_options_unstable(&icu_testdata::unstable(), options)
+        LineSegmenter::try_new_dictionary_with_options_unstable(&icu_testdata::unstable(), options)
             .expect("Data exists");
 
     group.bench_function("En", |b| {
```

Comment:
Suggestion: Please add a bench matrix for dictionary, lstm, and auto. Or, at least, change the bench names to have "dictionary" in them, and file a follow-up issue to add the full matrix.

Reply:
Sure. I've added a commit for the benchmarks in this PR.